### PR TITLE
[Gekidou] Update dependencies

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
Update dependencies

```release-note
NONE
```

### CODE REVIEW

**Review by josephbaylon — APPROVED** [review]
LGTM. Did some smoke testing and nothing seems to stand out on both iOS and Android

File: jest.config.js [hunk 1]
```diff
@@ -15,9 +15,8 @@ module.exports = {
     collectCoverageFrom: ['app/**/*.{js,jsx,ts,tsx}'],
     coverageReporters: ['lcov', 'text-summary'],
     testPathIgnorePatterns: ['/node_modules/'],
-    moduleNameMapper: {
-        'assets/images/video_player/(.*).png':
-            '<rootDir>/dist/assets/images/video_player/$1@2x.png',
+    transform: {
```
Comment (larkox) [inline jest.config.js:18]: Just out of curiosity, but this change seems to be two completely different things. Is it ok to remove the previous one and add this?

**Review by larkox — APPROVED** [review]
LGTM, just one question.

File: jest.config.js (hunk 1 above)
Comment (enahum) [reply jest.config.js:18]: right, so the previous values in the name mapper do not apply anymore as we don't have the video_player assets anymore as we are using the native video controls.

As for the transform, I took it from https://jestjs.io/docs/code-transformation which is needed after upgrading to jest 28

**Pull request merged**